packages/libs/client/src/signing/tests/signWithChainweaver.test.ts

```diff
@@ -16,25 +19,30 @@ describe('signWithChainweaver', () => {
   it('makes a call on 127.0.0.1:9467/v1/quickSign with transaction', async () => {
     (fetch as jest.Mock).mockResolvedValue({
       status: 200,
-      text: () => JSON.stringify({ signedResponse: true }),
+      text: () => JSON.stringify({ results: [] }),
       json: () => {},
     });
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const pactModule = Pact.modules as any;
+
     const unsignedCommand = (
       pactModule.coin.transfer('k:from') as ICommandBuilder<{
         GAS: [];
       }> &
         IPactCommand
-    )
-      .addCap('GAS', 'signer-key')
-      .setMeta({
-        sender: '',
-      })
-      .createTransaction();
-    const body = JSON.stringify({ reqs: [unsignedCommand] });
+    ).addCap('GAS', 'signer-key');
+
+    const { cmd, hash } = unsignedCommand.createCommand();
+    const sigs = unsignedCommand.sigs.reduce((sigs, sig, i) => {
+      const pubkey = unsignedCommand.signers[i].pubKey;
+      sigs[pubkey] =
+        sig === undefined ? null : sig.sig === undefined ? null : sig.sig;
```

Comment:
Can't we simplefy this to: `sig?.sig ?? null`?

Reply:
See comment above